### PR TITLE
feat: add global config parameters

### PR DESCRIPTION
**Pull request by nodegard**

### CODE REVIEW

File: cognite/extractorutils/__init__.py [hunk 1]
```diff
@@ -16,5 +16,5 @@
 Cognite extractor utils is a Python package that simplifies the development of new extractors.
 """
 
-__version__ = "5.0.1"
+__version__ = "5.0.2"
```
Comment (einarmo) [inline cognite/extractorutils/__init__.py:19]: As this is a non-breaking feature update it should be a semimajor version, so we should be on `5.1.0`

File: cognite/extractorutils/configtools/elements.py [hunk 2]
```diff
@@ -220,6 +236,7 @@ class CogniteConfig:
     data_set_external_id: Optional[str]
     extraction_pipeline: Optional[EitherIdConfig]
     timeout: TimeIntervalConfig = TimeIntervalConfig("30s")
+    global_config: GlobalConfig = field(default_factory=GlobalConfig)
```
Comment (einarmo) [inline cognite/extractorutils/configtools/elements.py:239]: While it's called `global_config` in the SDK, I'm not so sure about exposing that to users who don't necessarily even know what kind of software the extractor is. Perhaps something like `connection`?

File: schema/cognite_config.schema.json [hunk 3]
```diff
@@ -104,6 +104,59 @@
...
+        "global-config": {
+            "type": "object",
+            "description": "Configure authentication using OIDC tokens",
+            "properties": {
+                "disable-gzip": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Whether or not to disable gzipping of json bodies."
+                },
+                "disable-pypi-version-check": {
```
Comment (einarmo) [inline schema/cognite_config.schema.json:117]: This field isn't actually included in config, and shouldn't be.

**Review by einarmo — CHANGES_REQUESTED** [review]
Looks good, I have a few things I'd like changed.

File: schema/cognite_config.schema.json (hunk 3 above)
Comment (nodegard) [reply schema/cognite_config.schema.json:117]: Thanks for noticing, I'd added it before I noticed you were already setting that to a static `True` 👍

**Review by einarmo — APPROVED** [review]
LGTM. Thanks for doing this.

**Pull request merged**